stripe/src/main/java/com/stripe/android/view/CardFormView.kt

```diff
@@ -270,6 +270,11 @@ internal class CardFormView @JvmOverloads constructor(
 
         cardMultilineWidget.cvcEditText.imeOptions = EditorInfo.IME_ACTION_NEXT
         cardMultilineWidget.setBackgroundResource(R.drawable.stripe_card_form_view_text_input_layout_background)
+        cardMultilineWidget.cvcEditText.doAfterTextChanged { cvcText ->
+            if (cardMultilineWidget.brand.isMaxCvc(cvcText.toString())) {
+                postalCodeView.requestFocus()
```

Comment:
Do we still want to do this when the country doesn't have postal codes?

Reply:
nice catch, added a check for postal visibility